echopop/workflow/feat_hake.py

```diff
@@ -575,101 +603,403 @@
-    ds_proportions=ds_proportions,
+# ##################################################################################################
+# Consolidate the kriged abundance estimates into a single DataFrame table
+# ------------------------------------------------------------------------
+
+df_kriged_abundance_table = apportion.combine_population_tables(
+    population_table=dict_kriged_abundance_table,
+    table_names=["aged", "standardized_unaged"],
+    table_index=["length_bin"],
+    table_columns=["age_bin", "sex"],
+)
+
+# ##################################################################################################
+# Consolidate the kriged biomass estimates into a single DataFrame table
+# -----------------------------------------------------------------------
+
+df_kriged_biomass_table = apportion.combine_population_tables(
+    population_table=dict_kriged_biomass_table,
+    table_names=["aged", "standardized_unaged"],
+    table_index=["length_bin"],
+    table_columns=["age_bin", "sex"],
+)
+
+# ##################################################################################################
+# Redistribute the kriged abundance estimates
+# -------------------------------------------
+# THIS NEEDS TO BE FULLY IMPLEMENTED WITH TESTS, ETC.
+
+# Re-allocate the age-1 abundance estimates 
+df_kriged_abundance_table_noage1 = apportion.redistribute_population_table(
+    population_table=df_kriged_abundance_table,
+    exclusion_filter={"age_bin": [1]},
+    group_by=["sex"],
+)
+
+####################################################################################################
+# Redistribute the kriged biomass estimates
+# -----------------------------------------
+
+# Re-allocate the age-1 abundance estimates 
+df_kriged_biomass_table_noage1 = apportion.redistribute_population_table(
+    population_table=df_kriged_biomass_table,
+    exclusion_filter={"age_bin": [1]},
+    group_by=["sex"],
+)
+
+# ##################################################################################################
+# Instantiate stratified analysis
+# -------------------------------
+# Model parameters for stratified analysis initialization
+JOLLYHAMPTON_PARAMETERS = {
+    "transects_per_latitude": 5,
+    "strata_transect_proportion": 0.75,
+    "num_replicates": 1000,
+}
+
+jh = stratified.JollyHampton(JOLLYHAMPTON_PARAMETERS)
+
+# ##################################################################################################
+# Compute coefficient of variation (CV) and other estimators for transect biomass
+# -------------------------------------------------------------------------------
+
+# Run bootstrapping procedure
+jh.stratified_bootstrap(data_df=df_nasc_no_age1, 
+                        stratify_by=["geostratum_inpfc"], 
+                        variable="biomass")
+
+# Compute summary statistics for each stratum and overall survey
+transect_results = jh.summarize(ci_percentile=0.95, ci_method="t-jackknife")
```

Comment:
I think `.summarize` here is a good name, but the operations are quite different from the 3 `._summarize_*` methods. How about changing `._summarize_*` to something more verbose like `._compute_*_statistics`?